@xen-orchestra/web/src/components/DonutWithLegends.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div class="donut-with-legends">
+    <DonutChart :segments="donutSegments" :icon="icon" :max-value="maxValue" />
```

Comment:
You can use the prop shorthand:
```suggestion
    <DonutChart :segments="donutSegments" :icon :max-value="maxValue" />
```